includes/languages/english.php

```diff
@@ -336,6 +336,7 @@ function zen_date_raw($date, $reverse = false) {
   define('SUCCESS_ADDED_TO_CART_PRODUCT', 'Successfully added Product to the cart ...');
 // only for where multiple add to cart is used:
   define('SUCCESS_ADDED_TO_CART_PRODUCTS', 'Successfully added selected Product(s) to the cart ...');
+  define('FAILURE_ADDED_TO_CART_PRODUCTS', 'The selected Product(s) are not currently available for purchase...');
```

Comment:
I'm puzzled by the name of this new constant: `FAILURE_ADDED_TO_CART_PRODUCTS`

Perhaps `FAILED_TO_ADD_UNAVAILABLE_PRODUCTS` might be more intuitive?

Reply:
:+1: